CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Version 4.0.3 (Under development)
 
+### App Center
+
+#### iOS
+
+* **[Feature]** Use XCFramework format for the AppCenterReactNativeShared pod.
```

Comment:
Let's mention that it fixes Apple Silicon simulators (and probably some release builds)